build.gradle

```diff
@@ -954,6 +956,8 @@ project(':share') {
 
   dependencies {
     implementation project(':server-common')
+    implementation project(':clients')
+    implementation project(':coordinator-common')
```

Comment:
Isn't there is a cycle i.e. `coordinator-common` requires `server`, `server` requires `share` and `share` requires  `coordinator-common`?